### ISSUE TITLE
User Menu Override

### ISSUE THREAD

**Issue by benhaynes** [directus/app#1709]
We have a Navigation Override, but this does not control the User Menu. We should look into the same feature/format/interface but for the User Menu.

https://github.com/directus/app/issues/125
https://github.com/directus/app/issues/1575

Ref: https://stackoverflow.com/questions/56392419/hide-menu-items-for-users-with-custom-role

**Comment by benhaynes** [comment]
To achieve better clarity/visibility, we are now tracking feature requests within the [Feature Request](https://github.com/directus/api/projects/40) project board.

**This issue being closed does not mean it's not being considered.**

**Comment by 7jp** [comment]
Hey, just want to reheat it. I just need to hide few items in user menu depending on user role. 
I did some investigations, and it looks that a few little things might help to do work-around for this without touching core, and even help in Directus customisations at all.

If we added to some html tag, class name with user role like - "user-role-editor" add relevant classes to user menu links, or item numbers to lists then hiding them would be done simply in css.

Another thing,  if we stored app object in some global variable, then in script.js some simple modifications could be done directly. 

Additional classes look clean and transparent to whole app, the latter more dirty, however can be like swiss army knife in troubles.
